agriculture/agriculturecore/templates/dashboard.html

```diff
@@ -88,8 +88,10 @@ <h6 class="card-title">Weather station</h6>
                                                                     <td><span class="digi-icon-color fas fa-sun fa-2x"></span></td>
                                                                 </tr>
                                                                 <tr>
-                                                                    <td><span id="wind_speed"><i class="fas fa-circle-notch fa-spin"></i></span> km/h</td>
-                                                                    <td><span id="rain_diff"><i class="fas fa-circle-notch fa-spin"></i></span> L/m²</td>
+                                                                    <td><span id="wind_speed"><i class="fas fa-circle-notch fa-spin"></i></span> km/h (<span id="wind_direction"><i class="fas fa-circle-notch fa-spin"></i></span>)
+
```

Comment:
Either fill this space with the <span> tag contained in the <td> or put the </td> tag right after the </span> to avoid having this blank line.